code/game/jobs/job/civilian_chaplain.dm

```diff
@@ -303,7 +367,7 @@
 			if("revolution", "rev", "revolt")
 				B.name = "Down With Nanotrasen"
 				J = "Revolutionary"
-				deity_name = "Maximilien Robespierre" //From THE Revolution that defined the rest
+				deity_name = pick("Maximilien Robespierre", "Saul Alinsky")
```

Comment:
Wouldn't the result of this be that the chaplain wouldn't know his own deity's name without asking?